content/en/docs/concepts/containers/container-lifecycle-hooks.md

```diff
@@ -38,7 +38,7 @@ No parameters are passed to the handler.
 
 This hook is called immediately before a container is terminated due to an API request or management event such as liveness probe failure, preemption, resource contention and others. A call to the preStop hook fails if the container is already in terminated or completed state.
 It is blocking, meaning it is synchronous,
-so it must complete before the call to delete the container can be sent.
+so it must complete before the signal to stop the Container can be sent.
```

Comment:
```suggestion
so it must complete before the signal to stop the container can be sent.
```

I think the signal mentioned here is _not_ a call to the Kubernetes API server, and if that's correct then I would write “container” in lowercase to make that clear.

Reply:
I agree, I thought this file was just capitalizing it as a style thing. Thanks for the suggestion!